charts/linkerd2/templates/controller.yaml

```diff
@@ -94,6 +94,7 @@ spec:
         volumeMounts:
         - mountPath: /var/run/linkerd/config
           name: config
+      {{- $_ := set .Values.global.proxy "resources" (default .Values.global.proxy.resources .Values.controllerProxyResources) -}}        
```

Comment:
Please replace the leading `{{-` with just `{{` to avoid suppressing the line break above, which is also breaking the unit tests. This applies to all the template files you modified.

Reply:
But there is an additional linebreak at the end of the line 97. Thus I have to eat the one in the previous line.

Reply:
That makes sense, but for some reason having both suppressors was rendering an invalid template. Anyways after the last set of changes it's working fine :man_shrugging: